src/lib/commandBarConfigs/validators.ts

```diff
@@ -104,3 +104,48 @@ export const revolveAxisValidator = async ({
     return 'Unable to revolve with selected axis'
   }
 }
+
+export const loftValidator = async ({
+  data,
+}: {
+  data: { [key: string]: Selections }
+  context: CommandBarContext
+}): Promise<boolean | string> => {
+  if (!isSelections(data.selection)) {
+    return 'Unable to revolve, selections are missing'
+  }
+
+  // TODO: should this be part of canLoftSelection? And should we use that here?
+  const sectionIds = data.selection.graphSelections.flatMap((s) =>
+    s.artifact?.type === 'solid2D' ? s.artifact.pathId : []
+  )
```

Comment:
Happy to get feedback on this

Reply:
Is this robust to check if the user has non-solid2D's in their selection? Should the expected behavior be to still loft if that is the case?

If we don't want to allow that, I think you should add logic disallowing it with a similar error toast.

If we do want to allow that and just ignore non-solid2D selections, this is fine.

Reply:
@franknoirot Just making sure, are you saying that we could provide a better error message by detecting that a selection is made of, for instance, 2 solid2D and 1 face, and rejecting it based on that ground instead of letting it go to the engine call and failing then? (which won't pass this function still but could be less useful)

Reply:
Oh nice that makes sense

Reply:
Ok cool I'll add that check thanks!

Reply:
Added @franknoirot. Let me know how that looks. Ty